ui/v2.5/src/components/Settings/Tasks/LibraryTasks.tsx

```diff
@@ -74,17 +74,17 @@ export const LibraryTasks: React.FC = () => {
   const [configureDefaults] = useConfigureDefaults();
 
   const [dialogOpen, setDialogOpenState] = useState({
-    clean: false,
     scan: false,
     autoTag: false,
+    autoTagAlert: false,
     identify: false,
   });
 
   const [scanOptions, setScanOptions] = useState<GQL.ScanMetadataInput>({});
   const [autoTagOptions, setAutoTagOptions] =
     useState<GQL.AutoTagMetadataInput>({
-      performers: ["*"],
-      studios: ["*"],
+      performers: [],
+      studios: [],
       tags: ["*"],
```

Comment:
The reason for specifying tags only as the default should be documented here. Right now, it just looks arbitrary.